vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -208,6 +208,12 @@ impl AudioBackend for PwBackend {
 
             let params = &stream_params[stream_id as usize].params;
 
+            if let Some(stream) = stream_hash.get(&stream_id) {
+                stream.disconnect().expect("could not disconnect stream");
```

Comment:
Is panicking here instead of returning an error on purpose? (Is this error unrecoverable?)

Reply:
right, I think returning an error instead is better.